### PR TITLE
Improve DarkMode (TitleBar)

### PR DESCRIPTION
**Pull request by ozone10**
Make DarkMode title bar to apply on all dialog windows. (Now should be missing only message box dialog windows)

fix https://github.com/notepad-plus-plus/notepad-plus-plus/issues/10022
partially fix https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9958

### CODE REVIEW

**Comment by donho** [comment]
@ozone10 
Thank you for your PR.
However, I don't see the difference before & after applying your PR.
A screenshot of comparison will help.

**Comment by ozone10** [comment]
Note the dialog windows title bar and scrollbar in ascii panel
Notepad++ v8.1
image


This pull request
image

Now dark title bar is applied almost everywhere, I think it is missing only for windows created by MessageBox function

**Review by donho — COMMENTED** [review]
Make 2 PR 1 for Titlebar, another for ScrollBar.

**Review by donho — CHANGES_REQUESTED** [review]
Make 2 PR 1 for Titlebar, another for ScrollBar.

**Comment by ozone10** [comment]
They are using same function. Should I split the function `void setDarkThemeViaWinAPI(HWND hwnd, bool setTitle);`
to one with "bool true" and one with "bool false".

**Comment by ozone10** [comment]
Now only affect title bar.

**Comment by donho** [comment]
@ozone10 
Thank you, the code is more readable now.
However, there is a regression comes with this PR: while we are in full dark mode (ie. restart Notepad++ to have titlebar and scrollbar in black), then we switch in light mode, the titlebar doesn't turn light.
The current implementation turns the titlebar light without restarting Notepad++.

**Review by donho — APPROVED** [review]

**Review by donho — CHANGES_REQUESTED** [review]
Fix the regression that titlebar doesn't turn into light mode directly.

**Comment by donho** [comment]
@ozone10 
Almost...
image

File: PowerEditor/src/WinControls/ClipboardHistory/clipboardHistoryPanel.rc [hunk 1]
```diff
@@ -24,5 +24,5 @@ EXSTYLE WS_EX_TOOLWINDOW | WS_EX_WINDOWEDGE
 CAPTION "Clipboard History"
 FONT 8, "MS Sans Serif", 0, 0, 0x0
 BEGIN
-	LISTBOX         IDC_LIST_CLIPBOARD,50,44,78,120,LBS_OWNERDRAWFIXED | LBS_NOINTEGRALHEIGHT | WS_VSCROLL | WS_TABSTOP
+	LISTBOX         IDC_LIST_CLIPBOARD,50,44,78,120,LBS_OWNERDRAWFIXED | LBS_NOINTEGRALHEIGHT | WS_VSCROLL | WS_TABSTOP | NOT WS_BORDER
```
Comment (donho) [inline PowerEditor/src/WinControls/ClipboardHistory/clipboardHistoryPanel.rc:27]: Why do we need to remove `WS_BORDER` ?

Comment (donho) [reply PowerEditor/src/WinControls/ClipboardHistory/clipboardHistoryPanel.rc:27]: OK, I see the visual effect.

**Comment by donho** [comment]
@ozone10 
Now you can add Scrollbar PR please

**Pull request closed without merging**